<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color:#f0f0f0;
}
QCheckBox {
	padding:2px;
}
QCheckBox:hover {
	border:1px solid rgb(255,150,60);
	border-radius:4px;
	padding: 1px;
	background-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(190, 90, 50, 50), stop:1 rgba(250, 130, 40, 50));
}
QCheckBox::indicator:checked {
	border:1px solid rgb(246, 134, 86);
	border-radius:4px;
  	background-color:rgb(246, 134, 86)
}
QCheckBox::indicator:unchecked {
	border-width:1px solid rgb(246, 134, 86);
	border-radius:4px;
  	background-color:rgb(255,255,255);
}
QColorDialog {
	background-color:#f0f0f0;
}
QComboBox {
	color:rgb(81,72,65);
	background: #ffffff;
}
QComboBox:editable {
	selection-color:rgb(81,72,65);
	selection-background-color: #ffffff;
}
QComboBox QAbstractItemView {
	selection-color: #ffffff;
	selection-background-color: rgb(246, 134, 86);
}
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
	color:  #1e1d23;	
}
QDateTimeEdit, QDateEdit, QDoubleSpinBox, QFontComboBox {
	color:rgb(81,72,65);
	background-color: #ffffff;
}

QDialog {
	background-color:#f0f0f0;
}


QLineEdit {
	background-color:rgb(255,255,255);
	selection-background-color:rgb(236,116,64);
}
QMenuBar {
	color:rgb(223,219,210);
	background-color:rgb(65,64,59);
}
QMenuBar::item {
	padding-top:4px;
	padding-left:4px;
	padding-right:4px;
	color:rgb(223,219,210);
	background-color:rgb(65,64,59);
}
QMenuBar::item:selected {
	color:rgb(255,255,255);
	padding-top:2px;
	padding-left:2px;
	padding-right:2px;
	border-top-width:2px;
	border-left-width:2px;
	border-right-width:2px;
	border-top-right-radius:4px;
	border-top-left-radius:4px;
	border-style:solid;
	background-color:rgb(65,64,59);
	border-top-color: rgb(47,47,44);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(90, 87, 78, 255), stop:1 rgba(47,47,44, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(90, 87, 78, 255), stop:1 rgba(47,47,44, 255));
}
QMenu {
	color:rgb(223,219,210);
	background-color:rgb(65,64,59);
}
QMenu::item {
	color:rgb(223,219,210);
	padding:4px 10px 4px 20px;
}
QMenu::item:selected {
	color:rgb(255,255,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(225, 108, 54, 255), stop:1 rgba(246, 134, 86, 255));
	border-style:solid;
	border-width:3px;
	padding:4px 7px 4px 17px;
	border-bottom-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(175,85,48,255), stop:1 rgba(236,114,67, 255));
	border-top-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-right-color:qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-left-color:qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
}
QPlainTextEdit {
	border: 1px solid transparent;
	color:rgb(17,17,17);
	selection-background-color:rgb(236,116,64);
    background-color: #FFFFFF;
}
QProgressBar {
	text-align: center;
	color: rgb(0, 0, 0);
	border: 1px inset rgb(150,150,150); 
	border-radius: 10px;
	background-color:rgb(221,221,219);
}
QProgressBar::chunk:horizontal {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(225, 108, 54, 255), stop:1 rgba(246, 134, 86, 255));
	border:1px solid;
	border-radius:8px;
	border-bottom-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(175,85,48,255), stop:1 rgba(236,114,67, 255));
	border-top-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-right-color:qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-left-color:qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
}

QRadioButton {
	padding: 1px;
}
QRadioButton::indicator:checked {
	height: 10px;
	width: 10px;
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: rgba(246, 134, 86, 255);
	color: #a9b7c6;
	background-color:rgba(246, 134, 86, 255);
}
QRadioButton::indicator:!checked {
	height: 10px;
	width: 10px;
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: rgb(246, 134, 86);
	color: #a9b7c6;
	background-color: transparent;
}
QScrollArea {
	color: white;
	background-color:#f0f0f0;
}
QSlider::groove {
	border-style: solid;
	border-width: 1px;
	border-color: rgb(207,207,207);
}
QSlider::groove:horizontal {
	height: 5px;
	background: rgb(246, 134, 86);
}
QSlider::groove:vertical {
	width: 5px;
	background: rgb(246, 134, 86);
}
QSlider::handle:horizontal {
	background: rgb(253,253,253);
	border-style: solid;
	border-width: 1px;
	border-color: rgb(207,207,207);
	width: 12px;
	margin: -5px 0;
	border-radius: 7px;
}
QSlider::handle:vertical {
	background: rgb(253,253,253);
	border-style: solid;
	border-width: 1px;
	border-color: rgb(207,207,207);
	height: 12px;
	margin: 0 -5px;
	border-radius: 7px;
}
QSlider::add-page:horizontal, QSlider::add-page:vertical {
 	background: white;
}
QSlider::sub-page:horizontal, QSlider::sub-page:vertical {
	background: rgb(246, 134, 86);
}
QStatusBar, QSpinBox {
	color:rgb(81,72,65);
}
QSpinBox {
	background-color: #ffffff;
}
QScrollBar:horizontal {
	max-height: 20px;
	border: 1px transparent;
	margin: 0px 20px 0px 20px;
}
QScrollBar::handle:horizontal {
	background: rgb(253,253,253);
	border: 1px solid rgb(207,207,207);
	border-radius: 7px;
	min-width: 25px;
}
QScrollBar::handle:horizontal:hover {
	background: rgb(253,253,253);
	border: 1px solid rgb(255,150,60);
	border-radius: 7px;
	min-width: 25px;
}
QScrollBar::add-line:horizontal {
  	border: 1px solid rgb(207,207,207);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-right-radius: 7px;
  	background: rgb(255, 255, 255);
  	width: 20px;
  	subcontrol-position: right;
  	subcontrol-origin: margin;
}
QScrollBar::add-line:horizontal:hover {
  	border: 1px solid rgb(255,150,60);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-right-radius: 7px;
  	background: rgb(255, 255, 255);
  	width: 20px;
  	subcontrol-position: right;
  	subcontrol-origin: margin;
}
QScrollBar::add-line:horizontal:pressed {
  	border: 1px solid grey;
  	border-top-left-radius: 7px;
  	border-top-right-radius: 7px;
  	border-bottom-right-radius: 7px;
  	background: rgb(231,231,231);
  	width: 20px;
  	subcontrol-position: right;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
  	border: 1px solid rgb(207,207,207);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
  	background: rgb(255, 255, 255);
  	width: 20px;
  	subcontrol-position: left;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal:hover {
  	border: 1px solid rgb(255,150,60);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
  	background: rgb(255, 255, 255);
  	width: 20px;
  	subcontrol-position: left;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal:pressed {
  	border: 1px solid grey;
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
  	background: rgb(231,231,231);
  	width: 20px;
  	subcontrol-position: left;
  	subcontrol-origin: margin;
}
QScrollBar::left-arrow:horizontal {
  	border: 1px transparent grey;
  	border-top-left-radius: 3px;
  	border-bottom-left-radius: 3px;
  	width: 6px;
  	height: 6px;
  	background: rgb(230,230,230);
}
QScrollBar::right-arrow:horizontal {
	border: 1px transparent grey;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
  	width: 6px;
  	height: 6px;
 	background: rgb(230,230,230);
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
 	background: none;
} 
QScrollBar:vertical {
	max-width: 20px;
	border: 1px transparent grey;
	margin: 20px 0px 20px 0px;
}
QScrollBar::add-line:vertical {
	border: 1px solid;
	border-color: rgb(207,207,207);
	border-bottom-right-radius: 7px;
	border-bottom-left-radius: 7px;
	border-top-left-radius: 7px;
	background: rgb(255, 255, 255);
  	height: 20px;
  	subcontrol-position: bottom;
  	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {
  	border: 1px solid;
  	border-color: rgb(255,150,60);
  	border-bottom-right-radius: 7px;
  	border-bottom-left-radius: 7px;
  	border-top-left-radius: 7px;
  	background: rgb(255, 255, 255);
  	height: 20px;
  	subcontrol-position: bottom;
  	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:pressed {
  	border: 1px solid grey;
  	border-bottom-left-radius: 7px;
  	border-bottom-right-radius: 7px;
  	border-top-left-radius: 7px;
  	background: rgb(231,231,231);
  	height: 20px;
  	subcontrol-position: bottom;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
  	border: 1px solid rgb(207,207,207);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
  	background: rgb(255, 255, 255);
  	height: 20px;
  	subcontrol-position: top;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {
  	border: 1px solid rgb(255,150,60);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
	background: rgb(255, 255, 255);
  	height: 20px;
  	subcontrol-position: top;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:pressed {
  	border: 1px solid grey;
  	border-top-left-radius: 7px;
  	border-top-right-radius: 7px;
  	background: rgb(231,231,231);
 	height: 20px;
  	subcontrol-position: top;
  	subcontrol-origin: margin;
}
QScrollBar::handle:vertical {
	background: rgb(253,253,253);
	border: 1px solid rgb(207,207,207);
	border-radius: 7px;
	min-height: 25px;
}
QScrollBar::handle:vertical:hover {
	background: rgb(253,253,253);
	border: 1px solid rgb(255,150,60);
	border-radius: 7px;
	min-height: 25px;
}
QScrollBar::up-arrow:vertical {
	border: 1px transparent grey;
  	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
  	width: 6px;
  	height: 6px;
  	background: rgb(230,230,230);
}
QScrollBar::down-arrow:vertical {
  	border: 1px transparent grey;
  	border-bottom-left-radius: 3px;
  	border-bottom-right-radius: 3px;
  	width: 6px;
  	height: 6px;
  	background: rgb(230,230,230);
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
  	background: none;
}
QTabWidget {
	color:rgb(0,0,0);
	background-color:rgb(247,246,246);
}
QTabWidget::pane {
	border-color: rgb(180,180,180);
	background-color:rgb(247,246,246);
	border-style: solid;
	border-width: 1px;
  	border-radius: 6px;
}
QTabBar::tab {
	padding-left:4px;
	padding-right:4px;
	padding-bottom:2px;
	padding-top:2px;
	color:rgb(81,72,65);
  	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(221,218,217,255), stop:1 rgba(240,239,238,255));
	border-style: solid;
	border-width: 1px;
  	border-top-right-radius:4px;
	border-top-left-radius:4px;
	border-top-color: rgb(180,180,180);
	border-left-color: rgb(180,180,180);
	border-right-color: rgb(180,180,180);
	border-bottom-color: transparent;
}
QTabBar::tab:selected, QTabBar::tab:last:selected, QTabBar::tab:hover {
  	background-color:rgb(247,246,246);
  	margin-left: 0px;
  	margin-right: 1px;
}
QTabBar::tab:!selected {
	margin-top: 1px;
	margin-right: 1px;
}
QTextEdit {
	border-width: 1px;
	border-style: solid;
	border-color:transparent;
	color:rgb(17,17,17);
	selection-background-color:rgb(236,116,64);
}
QTimeEdit, QToolBox, QToolBox::tab, QToolBox::tab:selected {
	color:rgb(81,72,65);
	background-color: #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color:rgb(17,17,17);
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Реализация генетического алгоритма&lt;/p&gt;&lt;p&gt;размещения графа на линейке&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>true</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(17,17,17);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Состояние алгоритма: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>true</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>true</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Результат: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(230, 230, 230, 255));
}</string>
          </property>
          <property name="text">
           <string>Построить график</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(230, 230, 230, 255));
}</string>
          </property>
          <property name="text">
           <string>Запустить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Параметры</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/openFolder.ico</normaloff>:/openFolder.ico</iconset>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Загрузить файл графа&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/saveFolder.ico</normaloff>:/saveFolder.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/settings.ico</normaloff>:/settings.ico</iconset>
   </property>
   <property name="text">
    <string>Установка параметров</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/help.ico</normaloff>:/help.ico</iconset>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="resource/res.qrc">
     <normaloff>:/autors.ico</normaloff>:/autors.ico</iconset>
   </property>
   <property name="text">
    <string>Авторы</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource/res.qrc"/>
 </resources>
 <connections/>
</ui>
